1Password/docusaurus-extensions
# [DEV] Bump Node Version to 18.19.1

## Summary

Update the supported version of Node within our project.

## Related Issues and MRs

Closing #62 

## What changed

- Update Node version within `.nvmrc`
- Update CI

## Steps for reviewer

1. Pull changes and ensure deps are up to date
```
git checkout brian/mdx-3
yarn install
```
2. Using [nvm](https://github.com/nvm-sh/nvm) run the following (other Node version managers may have varied workflows): 
```
nvm use
```
3. Ensure plugins run properly
```
yarn dev docusaurus-plugin-stored-data
```
```
yarn dev jest-docusaurus
```

## Additional resources

- https://docusaurus.io/blog/preparing-your-site-for-docusaurus-v3#nodejs-180
